charts/DataTab.tsx

```diff
@@ -20,49 +21,59 @@ export class DataTab extends React.Component<{
     // Here's where the actual CSV is made
     @computed get csvBlob() {
         const { chart } = this.props
-        const { vardata } = chart
+        const yearIsDayVar = chart.yearIsDayVar
+        const dayIndexedCSV = yearIsDayVar ? true : false
 
         const dimensions = chart.data.filledDimensions.filter(
             d => d.property !== "color"
```

Comment:
This is unrelated to your commit but it surprises me that we'd filter this one out. My understanding is that most of the time color dimension is used it is used for  continents. Not sure what the harm is in including that field in the outputted CSV. As a user I'd expect all data used to make a chart would be available in the csv.

Reply:
@vanwoerden @jasoncrawford Thoughts? Easy fix.